### PR TITLE
All category

### PR DESCRIPTION
**Pull request by Avetiq30**
https://www.notion.so/B-Get-All-Category-6d26e0bea9b0438db54d03d8248667c7?pvs=4

### CODE REVIEW

File: src/category/category.service.ts [hunk 1]
```diff
@@ -17,7 +17,11 @@ export class CategoryService {
   }
 
   async getAllCategories(): Promise<any[]> {
-    return this.categoryModel.find().exec();
+    const category = await this.categoryModel.find().exec();
+    if (category.length === 0) {
+      return HttpStatus.OK, [];
+    }
+    return category;
```
Comment (VahagShahinyan) [inline src/category/category.service.ts:24]: replace ```return this.categoryModel.find().exec();```

File: src/user/user.service.ts [hunk 2]
```diff
@@ -25,6 +28,23 @@ export class UserService {
...
+    return this.createUser({ ...userData, role: 'user' });
+  }
+
+  async getAllUser(): Promise<UserModel[]> {
+    const users = await this.userModel.find().exec();
+
+    if (users.length === 0) {
+      return HttpStatus.OK, [];
+    }
+    return users;
```
Comment (VahagShahinyan) [inline src/user/user.service.ts:45]: ```return this.userModel.find().exec();```

File: src/category/category.service.ts (hunk 1 above)
Comment (Avetiq30) [reply src/category/category.service.ts:24]: ++

File: src/user/user.service.ts (hunk 2 above)
Comment (Avetiq30) [reply src/user/user.service.ts:45]: ++

**Pull request merged**